dependencies.properties

```diff
@@ -8,16 +8,16 @@
 
 # Versions of oneself
 # {x-version-update-start:gax:current}
-version.gax=1.55.0
+version.gax=1.56.0
 # {x-version-update-end}
 # {x-version-update-start:gax:current}
-version.gax_grpc=1.55.0
+version.gax_grpc=1.56.0
 # {x-version-update-end}
 # {x-version-update-start:gax:current}
-version.gax_bom=1.55.0
+version.gax_bom=1.56.0
 # {x-version-update-end}
 # {x-version-update-start:gax-httpjson:current}
-version.gax_httpjson=0.72.0
+version.gax_httpjson=0.73.0
```

Comment:
are we ever going to make this 1.0?

Reply:
This artifact is only used by a single, [alpha client](https://github.com/googleapis/java-compute) - it's for a gapic-style generated client from the discovery docs. This artifact is not stable yet - perhaps when more resources are available to work on that generator output, we can make it 1.0.